### PR TITLE
kops: 1.20.1 -> 1.20.2

### PR DESCRIPTION
**Pull request by yurrriq**
###### Motivation for this change

Update to latest version.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/cluster/kops/default.nix [hunk 1]
```diff
@@ -65,7 +65,7 @@ rec {
   };
 
   kops_1_20 = mkKops rec {
-    version = "1.20.1";
+    version = "1.20.2";
     sha256 = "sha256-k6ODXbh7Bh+rBw6bjSNLxLY0fz7JLnrmJibnXz5qnSc=";
```
Comment (zimbatm) [inline pkgs/applications/networking/cluster/kops/default.nix:69]: the hash also needs to be updated

Comment (yurrriq) [reply pkgs/applications/networking/cluster/kops/default.nix:69]: that's the value my machine gave me... I'll try again.

Comment (yurrriq) [reply pkgs/applications/networking/cluster/kops/default.nix]: For some reason I had to use `nix-prefetch-github` to get (presumably) the correct hash... Both work on my machine somehow.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 128801` at 05df2bdc run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/864e1a4f11a146b5ee6821c1d49f0682">kops</a> (kops_1_20)</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

**Comment by yurrriq** [comment]
Apparently something is off with my machine, so if someone else could test this and/or fix the hash, it would be greatly appreciated.

**Comment by SuperSandro2000** [comment]
@ofborg eval

File: pkgs/applications/networking/cluster/kops/default.nix [hunk 2]
```diff
@@ -65,8 +65,8 @@ rec {
   };
 
   kops_1_20 = mkKops rec {
-    version = "1.20.1";
-    sha256 = "sha256-k6ODXbh7Bh+rBw6bjSNLxLY0fz7JLnrmJibnXz5qnSc=";
+    version = "1.20.2";
+    sha256 = "1yafvimyl8xmzanmfl34ankkp0arggxiz49hsf8ds4hj3d68mn1f";
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/cluster/kops/default.nix:69]:
```suggestion
    sha256 = "011ib3xkj6nn7qax8d0ns8y4jhkwwmry1qnzxklvzssaxhmzs557";
```

**Pull request merged**